app/components/Documentation/components/Pagination/ButtonPaginationMobile.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import ArrowDownIcon from 'icons/ArrowDown';
+import { ButtonPaginationProps } from '../../types';
+import { NEXT_TYPE_BUTTON, PREV_TYPE_BUTTON } from './constants';
+
+const ButtonPaginationMobile: React.FC<ButtonPaginationProps> = ({
+  isStartArticle,
+  isLastArticle,
+  onClickPaginationArticle,
+  title,
+}): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <div
+      className="full-width pa l0 b0"
+      css={{
+        height: '60px',
+        border: '1px solid rgba(53, 74, 137, 0.15)',
+        backgroundColor: 'var(--color-white)',
```

Comment:
Same as above

Reply:
Also check other places and fix

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1385/commits/a7097e81a0e7d73ac85d3d914cf0d34be5df315d

Reply:
Confirm